src/bio/bio_monitor.c

```diff
@@ -373,10 +531,13 @@ get_spdk_log_page_completion(struct spdk_bdev_io *bdev_io, bool success,
 	bdev = spdk_bdev_desc_get_bdev(dev_health->bdh_desc);
 	D_ASSERT(bdev != NULL);
 
-	/* Store device health info in in-memory health state log. */
+	/* Store Intel SMART stats in in-memory health state log. */
+	if (dev_health->bdh_vendor_id != SPDK_PCI_VID_INTEL)
```

Comment:
How can this happen? we won't issue the command if vender id isn't intel, right?